### ISSUE TITLE
TypeError: Cannot read property 'url' of null

### ISSUE THREAD

**Issue by zeeshanjan82** [christopherthielen/ui-router-extras#193]
when I try to incorporate or directly run code from: https://github.com/kbdaitch/ui-router-lazy-example

  I get a strange error: TypeError: Cannot read property 'url' of null

  I have also attached the screenshot for the error. Is it possible for you to guide me why this error in coming.
![screen shot 2015-05-01 at 10 17 54 am](https://cloud.githubusercontent.com/assets/8342604/7426987/93f91556-efef-11e4-9d41-947c85ec66a7.png)

**Comment by christopherthielen** [comment]
Probably related to #167 
I've merged that in.  Try building ui-router-extras yourself from master:

```
git clone https://github.com/christopherthielen/ui-router-extras.git
cd ui-router-extras
npm install
gulp
```
you should have a built file in `build/ct-ui-router-extras.js`

**Comment by christopherthielen** [comment]
I think this is fixed in 0.0.14.  If not, please reopen.
